concourse/enumerator.py

```diff
@@ -88,6 +88,7 @@ def _wrap_into_descriptors(
         override_definitions={},
         job_mapping: model.concourse.JobMapping = None,
         target_team: str=None,
+        ref_sha: str = None,
```

Comment:
I think this name is a bit too short. how about `pipeline_definition_committish` (since this is to my understanding what it is - it is the commit hash pointing to the commit from which the pipelinedefinition was read..)